server/activate_hooks_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin/plugintest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestOnActivate(t *testing.T) {
```

Comment:
This is awesome! Great idea showcasing how to use the testing framework... and of course testing the plugin itself :D

Reply:
If you think it help, I would also work on adding more test cases.

Reply:
By all means!